frontend/packages/ux-editor/src/App.tsx

```diff
@@ -78,12 +76,6 @@ export function App() {
     }
   }, [setSelectedLayoutSet, selectedLayoutSet, layoutSets, app]);
 
-  if (!shouldDisplayFeature('shouldOverrideAppFrontendCheck')) {
```

Comment:
We should remove the feature flag from our `SupportedFeatureFlags` enum within `featureToggleUtils.ts`

```typescript
export type SupportedFeatureFlags =
  | 'componentConfigBeta'
  | 'configureLayoutSet'
  | 'shouldOverrideAppLibCheck'
  | 'resourceAccessLists'
  | 'shouldOverrideAppFrontendCheck'; <--- Remove this one
```

Reply:
Yes, I removed it along with its associated test